sample_apps/simple_app_under_oe/Makefile

```diff
@@ -23,6 +23,7 @@ cleanall: clean
 
 dump_mrenclave:
 	oesign dump --enclave-image=./enclave/enclave.signed
+	@echo "\e[1;31m Build measurement_init.exe with make -f cert_utility.mak measurement_init.exe in utilities directory\e[1;m"
 	@echo "\e[1;31m Invoke measurement_init.exe --mrenclave=xxx --out_file=binary_trusted_measurements_file.bin\e[1;m"
 	@echo "\e[1;31m Replace 'xxx' with the mrenclave string from the output above\e[1;m"
-	@echo "\e[1;31m Build measurement_init.exe with make -f cert_utility.mak measurement_init.exe in utilities directory\e[1;m"
+	tput init
```

Comment:
I had to add this to re-set the red-char font that the previous message leaves the terminal in.

Otherwise, in an interactive usage, the rest of the commands will be in red, which is distracting.

This fix works for interactive execution of this script. But if you redirect output to a file, then again, this `tput init` seems to "confuse" the redirection. 

I noticed that the execution sort-of 'halts' at this point.

I had to manually run `tput init` in that terminal for the background'ed execution to continue.

All this is messy ... and we need a cleaner solution for both requirements (of displaying the message in red font and then to reset the stty terminal after).

Any ideas?

Reply:
Have you tried `echo -e "\e[0mNormal Text"`?

Reply:
Essentially "\e[0m". The reset VT100 sequence? I can never remember these. See: https://m.blog.naver.com/PostView.naver?isHttpsRedirect=true&blogId=heehow&logNo=221603147435

Reply:
Adding an extra final echo line, like `@echo "\e[0;m"` seems to have resolved this stty color issue.

`./test.sh` now runs w/o hanging. Thanks for the tip!